packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -114,6 +92,61 @@ const SideNavItemContent = ({
    */
   scrollRef?: React.RefObject<ScrollIntoViewHandle>;
 }) => {
+  const sidenavItemContentContext = useSideNavItemContent();
+  const contextValue = useMemo(
+    () => sidenavItemContentContext,
+    [sidenavItemContentContext],
+  );
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+  const navItemContentStyles = useMemo(
+    () => ({
+      display: "flex",
+      alignItems: "center",
+      width: "100%",
+      textDecoration: "none",
+      color: `${isDisabled ? odysseyDesignTokens.TypographyColorDisabled : odysseyDesignTokens.TypographyColorHeading} !important`,
```

Comment:
Do we need `!important` here? All of these are local styles.

Reply:
yes, looks like we are hard-coding the color in the Mui overrides